### PR TITLE
802.1X documentation for HP Procurve 2610

### PR DESCRIPTION
**Pull request by lzammit**

### CODE REVIEW

**Comment by fdurand** [comment]
Check the assumptions in the beginning of the doc and change the values (like a.b.c.d => 192.168.1.5)

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 1]
```diff
@@ -1995,6 +1995,49 @@ Don't forget to permit address moves and the reauth period.  x represents the po
...
+
+   aaa port-access authenticator x
+   aaa port-access authenticator x client-limit 1
+   aaa port-access authenticator active
+   aaa port-access mac-based x
+   aaa port-access mac-based x addr-moves
+   aaa port-access mac-based x reauth-period 14400
+   aaa port-access x controlled-direction in
+
+(Thanks to Denis Bonnenfant for this contribution)
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc:2037]: Remove that line. It's not where we give credits.

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 2]
```diff
@@ -1995,6 +1995,49 @@ Don't forget to permit address moves and the reauth period.  x represents the po
    
 (Thanks to Jean-Francois Laporte for this contribution)
 
+2610
+^^^^
+
+802.1X
+++++++
+
+Assumption: We use x as the port name.
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Don't use x -> use 10 or something.

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 3]
```diff
@@ -1995,6 +1995,49 @@ Don't forget to permit address moves and the reauth period.  x represents the po
...
+
+Define the RADIUS server host:
+
+   radius-server host a.b.c.d key "verysecret"
+   radius-server host a.b.c.d acct-port 1813 key "verysecret"
```
Comment (fdurand) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: radius password: useStrongerSecret

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 4]
```diff
@@ -1995,6 +1995,47 @@ Don't forget to permit address moves and the reauth period.  x represents the po
...
+2610
+^^^^
+
+802.1X
+++++++
+
+Define the RADIUS server host:
+
+   radius-server host 192.168.1.5 key "verysecret"
+   radius-server host 192.168.1.5 acct-port 1813 key "verysecret"
```
Comment (fdurand) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Radius password: useStrongerSecret

**Comment by dwlfrth** [comment]
Looks like it is ready to be merged.

**Pull request merged**